source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -133,6 +133,13 @@ Read Preference
 connecting to the source and destination clusters. For more information,
 see :ref:`connections-read-preference`. 
 
+Legacy Index Value Rewrites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` does not support legacy index values, like ``0`` or an empty
```

Comment:
I feel "does not support" means that it's not allowed. But mongosync is actually handling legacy index values in this case.